### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by nikhillakshman**
Release Extensions 1.4.0

### CODE REVIEW

File: docs/interfaces/sheet.html [hunk 1]
```diff
@@ -117,7 +117,7 @@ <h3>name</h3>
 					<div class="tsd-signature tsd-kind-icon">name<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-type">string</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in SheetInterfaces.ts:147</li>
```
Comment (seanmakesgames) [inline docs/interfaces/sheet.html:120]: Interesting-- Do we know why the relative path disappeared off this text?

File: docs/enums/tableau.analyticsobjecttype.html [hunk 2]
```diff
@@ -295,7 +296,7 @@ <h2>Legend</h2>
                 m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
         })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-        ga('create', 'UA-625217-51', '');
+        ga('create', 'UA-625217-51', 'auto');
```
Comment (seanmakesgames) [inline docs/enums/tableau.analyticsobjecttype.html:299]: Do we know why this line in the google analytics code changed?

File: docs/enums/tableau.analyticsobjecttype.html [hunk 3]
```diff
@@ -87,30 +87,30 @@ <h2>Enumeration members</h2>
 				<section class="tsd-panel tsd-member tsd-kind-enum-member tsd-parent-kind-enum">
 					<a name="cluster" class="tsd-anchor"></a>
 					<h3>Cluster</h3>
-					<div class="tsd-signature tsd-kind-icon">Cluster<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-symbol"> =&nbsp;&quot;cluster&quot;</span></div>
+					<div class="tsd-signature tsd-kind-icon">Cluster<span class="tsd-signature-symbol">:</span> <span class="tsd-signature-symbol"> = &quot;cluster&quot;</span></div>
 					<aside class="tsd-sources">
 						<ul>
-							<li>Defined in Namespaces/Tableau.ts:26</li>
+							<li>Defined in src/ExternalContract/Namespaces/Tableau.ts:26</li>
```
Comment (seanmakesgames) [inline docs/enums/tableau.analyticsobjecttype.html:93]: Do we know why the relative paths here were added?

Comment (d45) [inline docs/enums/tableau.analyticsobjecttype.html:93]: @nikhillakshman @johnDance   Does it matter if we show the src path in the reference page?  Shouldn't we just show `Namespaces/Tableau.ts` ?

Comment (seanmakesgames) [reply docs/enums/tableau.analyticsobjecttype.html:93]: If you don't know where this came from either, it's probably a configuration change in docs generation-
Do either of you know? john, nikhil

File: docs/enums/tableau.analyticsobjecttype.html (hunk 2 above)
Comment (d45) [reply docs/enums/tableau.analyticsobjecttype.html:299]: Interesting.  It doesn't look like we changed anything in the typedoc commnad in  package.json.  The  `auto` might refer to the GA site. WE don't use it, but the `-gaSite` option defaults to `auto`.

Comment (seanmakesgames) [reply docs/enums/tableau.analyticsobjecttype.html:299]: Typedoc version was updated, I believe. These must just be new defaults.

Comment (d45) [reply docs/enums/tableau.analyticsobjecttype.html:299]: A Typedoc update might explain the relative paths being added too.

File: docs/enums/tableau.analyticsobjecttype.html (hunk 3 above)
Comment (d45) [reply docs/enums/tableau.analyticsobjecttype.html:93]: Looks like it picks up that path in the Typdoc build command.  
From package.json: 
`--out docs-extensions src/ExternalContract`  

I think `docs-extensions` is the name of the directory the HTML files get dumped in. Not sure about `/scr/ExternalContract`  - could be the path to files to process.

File: docs/interfaces/sheet.html (hunk 1 above)
Comment (johnDance) [reply docs/interfaces/sheet.html:120]: TypeDoc now adds the path from the root in the "Defined in" statement.

File: docs/enums/tableau.analyticsobjecttype.html (hunk 3 above)
Comment (johnDance) [reply docs/enums/tableau.analyticsobjecttype.html:93]: TypeDoc now adds the path from the root in the "Defined in" statement.

Comment (johnDance) [reply docs/enums/tableau.analyticsobjecttype.html:93]: TypeDoc now adds the path from the root in the "Defined in" statement.

Comment (seanmakesgames) [reply docs/enums/tableau.analyticsobjecttype.html:93]: Great. Thanks for looking that up.
I think we just need to decide what we want this to look like and when. Let's do that in the slack thread then update here.

**Comment by ashwinar** [comment]
We should also check-in the package-lock.json file that gets updated when you update types.

File: package.json [hunk 4]
```diff
@@ -21,7 +21,7 @@
     "http-server": "^0.11.1"
   },
   "devDependencies": {
-    "@tableau/extensions-api-types": "1.3.1",
+    "@tableau/extensions-api-types": "1.4.0",
```
Comment (ashwinar) [inline package.json:24]: also, I found one of the reasons why the update command wasn't working. It was missing the "^" in the version. 
Could you change the version to "^1.4.0"?

**Pull request merged**